routes/admin.js

```diff
@@ -1,6 +1,8 @@
 var express = require("express");
 var router = express.Router();
 var admin = require("../controls/admin");
+const Question = require("../models/problems");
+const TC = require("../models/testcases");
 
```

Comment:
Also, then you don't need to require problems and testcases (Line 4 and 5).

Reply:
Then I won't be able to fetch the question and testcase from the database

Reply:
Why not. It's working perfectly fine.
![a](https://user-images.githubusercontent.com/20883811/51922996-19a79300-2410-11e9-8354-c49364388465.png)
![b](https://user-images.githubusercontent.com/20883811/51922999-1a402980-2410-11e9-8901-a728ccf0e46a.png)

Reply:
Okay , previously I saw your comment to shift to editQuestion(put) so at that time it was required,
You're right,
If I also shift the function to handle the request getQuestion then It won't be required,
I'll do it